packages/fxa-auth-server/lib/routes/password.ts

```diff
@@ -598,6 +598,69 @@ module.exports = function (
         return {};
       },
     },
+    {
+      method: 'POST',
+      path: '/password/forgot/verify_otp',
+      options: {
+        ...PASSWORD_DOCS.PASSWORD_FORGOT_VERIFY_OTP_POST,
+        validate: {
+          payload: isA.object({
+            email: validators
+              .email()
+              .required()
+              .description(DESCRIPTION.emailRecovery),
+            code: isA
+              .string()
+              .length(config.passwordForgotOtp.digits)
+              .regex(validators.DIGITS),
+            metricsContext: METRICS_CONTEXT_SCHEMA,
+          }),
+        },
+      },
+      handler: async function (request: any) {
```

Comment:
Why not make it `AuthRequest` like the rest?

Reply:
The request doesn't require auth though.  And the `request.payload` type is not compatible.  (Moreover, the metrics related types, which is what I really need in this handler, are [all anys](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/types.ts#L56-L61) anyway.

I can define a new type for that request though if that works for you.  (In addition to the metrics related types linked above, `db` is also typed as any, and so are many of the types that are accepted through the exported function.  So if we truly want to correctly type these endpoints, it'd take more than the scope of this work.)

Reply:
@bbangert haha sorry I'm not sure if your thumbs-up mean "I'm good with that explanation" or "yes, add a new type for the request" 😅

Reply:
Doh, sorry about that. I'm good with that explanation.